code/modules/projectiles/guns/ballistic/revolver.dm

```diff
@@ -151,7 +151,7 @@
 
 /obj/item/gun/ballistic/revolver/nagant
 	name = "\improper Nagant revolver"
-	desc = "An old model of revolver that originated in Russia. Able to be suppressed. Uses 7.62x38mmR ammo."
+	desc = "An old model of revolver that originated in Russia. Has a threaded barrel for supressors. Uses 7.62x38mmR ammo." //Supressor indication consistency
```

Comment:
Nagant revolvers are able to be suppressed cause they have a mechanism to seal the chamber, not just cause they have a threaded barrel.